torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -41,33 +40,23 @@ def extra_repr(self) -> str:
 
 
 class ToImageTensor(Transform):
-
-    # Updated transformed types for ToImageTensor
-    _transformed_types = (is_simple_tensor, features._Feature, PIL.Image.Image, np.ndarray)
+    _transformed_types = (is_simple_tensor, features.Image, PIL.Image.Image, np.ndarray)
```

Comment:
Why `features.Image` here? The whole point is to transform into `features.Image` so shouldn't we just leave it be?

Reply:
We don't want a BBox to be turned into a `feature.Image`. The `_transform()` checked for `isinstance(inpt, (features.Image, PIL.Image.Image, np.ndarray)) or is_simple_tensor(inpt)`, so here I align it with this behaviour. Let me know if you see something weird.

Reply:
The same argument can be made for `ToImagePIL` below. I maintained this because of the `mode` changing feature but perhaps we shouldn't allow that either. Thoughts?

Reply:
We spoke offline and decided to remove them.